.stylelintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  extends: 'stylelint-config-standard-scss',
+  rules: {
+    'custom-property-pattern': null,
```

Comment:
We have a mixture of camelCase and kebab case rules in our styles thanks to our own CSS module classes, targeting 3rd party components like `tippy`, and VSCode variables using a mixture of both. I disabled pattern rules, but we could also set things for `camelCase` and disable certain files.

Reply:
we can't control other packages but I think we use camelCase, right? We should turn on and exclude files/lines where we can't update. It will probably encourage us to wrap external modules in specific places instead of using them all throughout the code.

Reply:
All kebab-case rules should be inside `:global(...)`. Looking at stylelint configure docs, I've stumbled upon this:

```
{
  "rules": {
    "selector-pseudo-class-no-unknown": [
      true,
      {
        "ignorePseudoClasses": ["global"]
      }
    ]
  }
}
```

Not sure if it applies in this case, but if we can ignore global we could set the rule correctly to camelCase.

On another note (out of scope), unless it's really specific to the current CSS module (`.myModuleBlock :global(.lib-class) {`), we should move the `:global(....)` selector to the `shared/style.scss` file without using `global` and probably ignore this whole file.